src/pages/networks/actions/DeleteLocalPeerBtn.tsx

```diff
@@ -0,0 +1,92 @@
+import type { FC } from "react";
+import { useState } from "react";
+import { useQueryClient } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import {
+  ConfirmationButton,
+  Icon,
+  useNotify,
+  useToastNotification,
+} from "@canonical/react-components";
+import { useCurrentProject } from "context/useCurrentProject";
+import ResourceLabel from "components/ResourceLabel";
+import type { LxdNetwork } from "types/network";
+import { deleteLocalPeering } from "api/network-local-peering";
+import { useNetworkEntitlements } from "util/entitlements/networks";
+
+interface Props {
+  network: LxdNetwork;
+  localPeering: string;
+}
+
+const DeleteLocalPeerBtn: FC<Props> = ({ network, localPeering }) => {
+  const notify = useNotify();
+  const [isLoading, setLoading] = useState(false);
+  const queryClient = useQueryClient();
+  const { canEditNetwork } = useNetworkEntitlements();
+  const { project } = useCurrentProject();
+  const projectName = project?.name || "";
+  const toastNotify = useToastNotification();
+
+  const onFinish = () => {
+    toastNotify.success(
+      <>
+        Local peering {localPeering} deleted for network{" "}
```

Comment:
Please use a `ResourceLabel` for the peering as well. We need to introduce a new type for it in the resource label and resource icon.